### PR TITLE
docs: add github repo link field

### PR DESCRIPTION
**Pull request by krshkun**
closes #ISSUE-NO

#### What does this PR do?

Context: Live stream https://www.youtube.com/watch?v=jWqMuVKU53c

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/portfolio-review.yml [hunk 1]
```diff
@@ -13,6 +13,13 @@ body:
       placeholder: "e.g.: https://<your-username>.github.io/portfolio"
     validations:
       required: true
+  - type: input
+    id: link
+    attributes:
+      label: GitHub Repo Link
+      placeholder: "e.g.: https://github.com/YOUR_USERNAME/REPO"
+    validations:
+      required: true
```
Comment (Panquesito7) [inline .github/ISSUE_TEMPLATE/portfolio-review.yml:22]: Do we need to make this required? This is for the portfolio review. 🤔

File: .github/ISSUE_TEMPLATE/portfolio-review.yml [hunk 2]
```diff
@@ -13,6 +13,13 @@ body:
       placeholder: "e.g.: https://<your-username>.github.io/portfolio"
     validations:
       required: true
+  - type: input
+    id: link
```
Comment (eddiejaoude) [inline .github/ISSUE_TEMPLATE/portfolio-review.yml:17]: `id` needs to be unique

```suggestion
    id: repo
```

File: .github/ISSUE_TEMPLATE/portfolio-review.yml (hunk 1 above)
Comment (eddiejaoude) [reply .github/ISSUE_TEMPLATE/portfolio-review.yml:22]: I think required is important 👍  I only want to review open source portfolios

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

**Review by eddiejaoude — APPROVED** [review]
Great team work 🔥

**Pull request merged**